templates/partners/desktop.html

```diff
@@ -7,143 +7,171 @@
+    </p>
+  </div>
+</section>
+
+
+<section class="p-strip has-accent-border--top-left is-deep">
+  <div class="row" style="margin-bottom: 3rem;">
+    <h2>Buy a certified Ubuntu PC</h2>
+    <p>We've been working with partners like Dell, HP and Lenovo for years to ensure that Ubuntu devices reach market as effectively as possible. With the help of these partners, Ubuntu machines are distributed around the world and can be found in both their exclusive and partner stores.</p>
+  </div>
+  <div class="row">
+    <div class="col-4 p-card has-cta">
+      <div class="p-card__content">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/d258d23b-2018-logo-dell.svg",
+          alt="Dell",
+          width="80",
+          height="80",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "u-hide--small u-hide--medium"}
+          ) | safe
+        }}
+        <p>Canonical and Dell work together to deliver Ubuntu on systems from entry-level laptops to high-end developer machines.</p>
+        <p><a href="https://www.delltechnologies.com/en-gb/ai-technologies/index.htm">See the Ubuntu certified AI-ready workstations&nbsp;&rsaquo;</a></p>
+      </div>
+      <a href="http://www.dell.com/linux" class="p-button--positive p-card__cta"><span>Shop for Dell with Ubuntu</span></a>
+    </div>
+    <div class="col-4 p-card has-cta">
+      <div class="p-card__content">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/f627fbfb-hp.svg",
+          alt="HP",
+          width="80",
+          height="80",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "u-hide--small u-hide--medium"}
+          ) | safe
+        }}
+        <p>HP sells a wide range of Canonical-certified Ubuntu laptops and desktop machines globally.</p>
+        <p><a href="">See the Ubuntu certified AI-ready workstations&nbsp;&rsaquo;</a></p>
+      </div>
+      <a href="https://certification.ubuntu.com/desktop/models?vendors=HP" class="p-button--positive p-card__cta"><span>View HP with Ubuntu</span></a>
+    </div>
+    <div class="col-4 p-card has-cta">
+      <div class="p-card__content">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/3180a6b5-partner-logo-lenovo.svg",
+          alt="Lenovo",
+          width="195",
+          height="40",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "u-hide--small u-hide--medium"}
+          ) | safe
+        }}
+        <p>Canonical has been shipping Ubuntu on a variety of Lenovo's popular machines which are available from Lenovo's enterprise teams.</p>
+        <p><a href="https://www.lenovo.com/us/en/think-workstations/artificial-intelligence-industry/c/artificial-intelligence-industry">Buy AI workstations with Ubuntu&nbsp;&rsaquo;</a></p>
+      </div>
+      <a href="https://certification.ubuntu.com/desktop/models?vendors=Lenovo" class="p-button--positive p-card__cta"><span>View Lenovo with Ubuntu</span></a>
```

Comment:
Links to https://ubuntu.com/certified?category=Desktop&category=Laptop on the doc

Reply:
I wasn't sure about these as they reference specific brands of laptop but want to link to the generic laptop page. I will ask in the doc.